apps/users/const.py

```diff
@@ -58,6 +58,18 @@ class KeyboardLayout(TextChoices):
     TR_TR_QWERTY = 'tr-tr-qwerty', 'Turkish-Q (Qwerty)'
     ES_ES_QWERTY = 'es-es-qwerty', 'Spanish'
     ES_LATAM_QWERTY = 'es-latam-qwerty', 'Spanish (Latin American)'
+    DE_DE_QWERTZ = 'de-de-qwertz', 'German (Qwertz)'
+    DE_CH_QWERTZ = 'de-ch-qwertz', 'Swiss German (Qwertz)'
+    SV_SE_QWERTY = 'sv-se-qwerty', 'Swedish (Qwerty)'
+    IT_IT_QWERTY = 'it-it-qwerty', 'Italian (Qwerty)'
+    PT_PT_QWERTY = 'pt-pt-qwerty', 'Portuguese (Qwerty)'
+    PT_BR_QWERTY = 'pt-br-qwerty', 'Portuguese Brazilian (Qwerty)'
+    PL_PL_QWERTY = 'pl-pl-qwerty', 'Polish (Qwerty)'
+    HU_HU_QWERTZ = 'hu-hu-qwertz', 'Hungarian (Qwertz)'
+    NO_NO_QWERTY = 'no-no-qwerty', 'Norwegian (Qwerty)'
+    RO_RO_QWERTY = 'ro-ro-qwerty', 'Romanian (Qwerty)'
+    DA_DK_QWERTY = 'da-dk-qwerty', 'Danish (Qwerty)'
+    FR_CA_QWERTY = 'fr-ca-qwerty', 'Canadian French (Qwerty)'
 
 
 class AppletConnectionMethod(TextChoices):
```

Comment:
I need more information to check the code differences between 2021 and 2024 versions specifically related to keyboard layout translations. Please provide the file name or path, along with a complete code snippet for comparison analysis.

For example: 
@@ -...
+      ...
@...-...

Without these details, I'll be unable to properly review the differences across different years' implementations of the Python project mentioned in your request.